README.md

```diff
@@ -1 +1,85 @@
-# book-dirt-bike-front-end
\ No newline at end of file
+# Dirt Bikes
```

Comment:
- [ ] When creating a rental, this happens:

![image](https://user-images.githubusercontent.com/57421823/193097833-8796b722-dd29-4c34-b4df-067f37f47df2.png)

Also, a message appears saying that the rental couldn't be created.

Kindly squash this bug, so your app works as intended 💪

Reply:
Thanks for your your review @Meltrust. We have fixed it.